src/__snapshots__/client.test.js.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Snapshots are a great way to enforce tests without manually writing out all the html in each test!